testdata/configtest.json

```diff
@@ -4,10 +4,31 @@
  "globalHTTPTimeout": 15000000000,
  "logging": {
   "enabled": true,
-  "file": "debug.txt",
-  "colour": false,
-  "level": "DEBUG|WARN|INFO|ERROR|FATAL",
-  "rotate": true
+  "advancedSettings": {
```

Comment:
Please revert this to master, but with the logger changes as this config is kept in sync with master to test backwards compatibility between engine and master